runtime/oti/jvminit.h

```diff
@@ -466,6 +466,8 @@ enum INIT_STAGE {
 #define VMOPT_XXCOMPACTSTRINGS "-XX:+CompactStrings"
 #define VMOPT_XXNOCOMPACTSTRINGS "-XX:-CompactStrings"
 
+#define VMOPT_XXALWAYSIPROFILEDURINGSTARTUPPHASE "-XX:alwaysIProfileDuringStartupPhase"
```

Comment:
XX options should have + and - in front of them. This allows user to enable/disable the feature even if some script already specifies it.